challenges/fullstack/challenge.md

```diff
@@ -19,148 +14,124 @@ We would like to evaluate your skill in the following areas:
 
 ## Levels
 
-There are 6 engineering levels at Teleport. Levels 1-4 are open through the
-fullstack challenge.
+There are 6 engineering levels at Teleport. Levels 1-4 are open to external candidates.
 
 For fullstack engineers, levels 5 and 6 are only for internal promotions. Check
 [Fullstack Engineering Levels](../../levels/fullstack.pdf) for more details.
 
-## Requirements
+At this time, we are seekling level 4 full stack engineers only.
+
+## Requirements - Level 4
 
 Implement an application that allows a user to browse directory content on a
 remote server.
 
 This application should have the following functionality:
 
-* A directory viewer client with client-side filtering and sorting capabilities
-  (level 1) and url-based navigation (level 2).
-* A server for serving the webassets, containerized using Docker (level 2).
-* Authentication with a login screen and backend API to browse a directory
-  (level 3+).
-
-### Level 1
-
-This level focuses on JS/CSS/DOM knowledge and does not require any server-side
-knowledge to implement.
-
-Create a directory viewer with the following functionality:
-
-* An interface for viewing and browsing directory content.
-* Filtering on filename within the current directory (basic string matching).
-* Sorting on filename, type, and size within the current directory.
-* Unlike simlar interfaces such as Google Drive and Dropbox, **file preview is
-  _not_ required**. Clicking on a file should be a no-op.
-
-Directions:
-
-*  Build your own components.
-*  Display the `name`, `size`, and `type` file attributes.
-*  Use human readable format to display the `size` of a file.
-*  Use below JSON structure to mock real directory.
-
-```js
-const directory = {
-  name: "teleport",
-  size: 0, // bytes
-  type: "dir",
-  items: [
-    {
-      name: "lib",
-      size: 0, // bytes
-      type: "dir",
-      items: [
-        {
-          name: "teleport.go",
-          size: 320000, // bytes
-          type: "file",
-        },
-        {
-          name: "test.go",
-          size: 3320000, // bytes
-          type: "file",
-        },
-      ],
-    },
-    {
-      name: "README.md",
-      size: 4340000, // bytes
-      type: "file",
-    },
-  ],
-};
-```
-
-You can use a design of similar viewers like Google Drive or Github as a
-starting point and then add missing pieces to it. For example, you can borrow
-CSS from these design systems:
-[Github](https://primer.style/components/getting-started) or
-[Google](https://material-ui.com/getting-started/usage/) if you like.
+* A Go backend that serves the webapp and an API
+* The UI, which should include client-side filtering and sorting capabilities
+  and URL-based navigation.
+* Strong authentication
 
-<img src="./assets/google.jpg" height="300" />
-<img src="./assets/github.jpg" height="200" />
+### API
 
-### Level 2
+This repository includes just enough starter code to serve up both the web app
+and a sample API endpoint.
```

Comment:
Where in the careers repo is the starter code?

Reply:
I'll clean this up. It's a separate repo.